mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po

```diff
@@ -3438,3 +3476,74 @@ msgstr ""
 
 msgid "spyder_flower_sandy_notweesher"
 msgstr "Oh, no importa. Veo que no tienes un Tweesher."
+
+msgid "spyder_route2_roddick1"
+msgstr "Cuando fui emboscado en la duna por SPIGHTER estuve muy asustado."
+
+msgid "spyder_route2_graf2"
+msgstr "Me sirvieron."
+
+msgid "spyder_wayfarer1_bismuth2"
+msgstr "¡Gorrón!"
+
+msgid "spyder_route2_roddick2"
+msgstr "Oye, ¡no parecías arrastrado en absoluto!"
+
+msgid "menu_down_key"
+msgstr "Tecla Bajar"
+
+msgid "menu_secondary_select_key"
+msgstr "Tecla Cancelar"
+
+msgid "spyder_route4_wulf2"
+msgstr "Ooh, ¡me siento tan castrado!"
+
+msgid "menu_left_key"
+msgstr "Tecla Izquierda"
+
+msgid "dark_robo"
+msgstr "Robo Oscuro"
+
+msgid "spyder_citypark_maniac"
+msgstr "Phew, ¡lo hemos cagado!"
+
+msgid "dodge_increased"
+msgstr "el {stat} de {user} ha incrementado."
+
+msgid "menu_up_key"
+msgstr "Tecla Subir"
+
+msgid "menu_right_key"
+msgstr "Tecla Derecha"
+
+msgid "menu_primary_select_key"
+msgstr "Tecla Acción"
+
+msgid "menu_back_key"
+msgstr "Tecla Volver"
+
+msgid "options_new_input_key0"
+msgstr "Pulse una tecla para fijar su tecla de entrada nueva"
+
+msgid "options_new_input_key1"
+msgstr "Reinicie su juego para que los cambios surtan en efecto."
+
+msgid "xeon"
+msgstr "Xeón"
+
+msgid "spyder_route4_wulf1"
+msgstr "Ah, tan solo recoger algunos suplementos medicinales"
+
+msgid "spyder_wayfarer1_bismuth1"
+msgstr ""
+"¡Espero que no esté relacionado a aquellos tuxemon enfermos en las granjas "
+"Scoop!"
+
+msgid "hp_restored"
+msgstr "el hp del {user} ha sido recuperada"
```

Comment:
HP is hit points. In Spanish it would be "puntos de golpe" or simply "salud" (health). You also are mixing genders here.
```suggestion
msgstr "La salud de {user} ha sido restaurada."
```